modules/tinymce/src/core/main/ts/newline/InsertBlock.ts

```diff
@@ -417,7 +434,7 @@ const insert = (editor: Editor, evt?: EditorEvent<KeyboardEvent>) => {
   }
 
   // Find parent block and setup empty block paddings
-  parentBlock = dom.getParent(container, dom.isBlock);
+  parentBlock = editor.dom.getParent(container, editor.dom.isBlock);
```

Comment:
Any particular reason why this was changed this will no longer name mangle producing a slightly bigger minified js file?

Reply:
I was changing it manually, and missed this detail, that's all.